src/components/elements/wrapper/wrapper.tsx

```diff
@@ -38,31 +38,40 @@ export const Wrapper = (props: IWrapperProps): JSX.Element | null => {
 		const wrapperChildren = schemaChildren || children;
 		if (typeof wrapperChildren === "object") {
 			const fieldTypeKeys = Object.keys(EFieldType);
+			const elementTypeKeys = Object.keys(EElementType);
 			const fieldComponents: JSX.Element[] = [];
 
 			Object.entries(wrapperChildren).forEach(([id, child]) => {
 				if (isEmpty(child)) return;
-
 				const fieldType = child.fieldType?.toUpperCase();
-
-				if (typeof child === "object" && fieldTypeKeys.includes(fieldType)) {
+				if (typeof child === "object") {
 					const frontendEngineComponents = { ...FrontendEngineFields, ...FrontendEngineElements };
-					const Field = (frontendEngineComponents[EFieldType[fieldType]] ||
-						Wrapper) as React.ForwardRefExoticComponent<IGenericFieldProps<TFrontendEngineFieldSchema>>;
-
-					fieldComponents.push(
-						<ConditionalRenderer id={id} key={id} renderRules={child.showIf}>
-							<Controller
-								control={control}
-								name={id}
-								shouldUnregister={true}
-								render={({ field, fieldState }) => {
-									const fieldProps = { ...field, id, ref: undefined }; // not passing ref because not all components have fields to be manipulated
-									return <Field schema={child} {...fieldProps} {...fieldState} />;
-								}}
-							/>
-						</ConditionalRenderer>
-					);
+					if (fieldTypeKeys.includes(fieldType)) {
```

Comment:
- differentiate between fields and elements
- only wrap fields in `Controller` to register them into react-hook-form
- elements don't have values so they don't need to be wrapped in `Controller`